lib/hyku_addons/engine.rb

```diff
@@ -534,7 +535,10 @@ def self.dynamically_include_mixins
       Hyrax::CurationConcern.actor_factory.insert_before Hyrax::Actors::ModelActor, HykuAddons::Actors::DateFieldsActor
       Hyrax::CurationConcern.actor_factory.insert_before Hyrax::Actors::ModelActor, HykuAddons::Actors::NoteFieldActor
       Hyrax::CurationConcern.actor_factory.insert_before Hyrax::Actors::ModelActor, HykuAddons::Actors::RelatedIdentifierActor
-      Hyrax::CurationConcern.actor_factory.insert_after HykuAddons::Actors::JSONFieldsActor, HykuAddons::Actors::CreatorInstitutionalEmailActor
+
+      actors = [HykuAddons::Actors::JSONFieldsActor, HykuAddons::Actors::CreatorProfileVisibilityActor]
+      Hyrax::CurationConcern.actor_factory.insert_after(*actors)
```

Comment:
@BertZZ The previous line was really long, so this is a nice way of breaking that down into an array and passing that to the `insert_after` method, where the array of extracted into arguments.